core/domain/collection_services.py

```diff
@@ -174,7 +212,15 @@ def get_collection_by_id(collection_id, strict=True, version=None):
 
 
 def get_collection_summary_by_id(collection_id):
-    """Returns a domain object representing a collection summary."""
+    """Returns a domain object representing a collection summary.
+
+    Args:
+        collection_id: Id of the collection summary.
+
+    Returns:
+        A domain object representing a collection summary of
```

Comment:
FYI, here, you can say something like:

    CollectionSummary. The collection summary domain object corresponding to the given collection_id.

(in case you're wondering what the 'type' of this return value is).